projects/media-viewer/src/lib/store/reducers/icp.reducer.ts

```diff
@@ -12,6 +12,15 @@ export function icpReducer (state = initialIcpSessionState,
 
   switch (action.type) {
 
+    case fromIcpActions.SET_CASE_ID: {
+      const caseId = action.payload;
+      const session = {...state.session, caseId};
```

Comment:
wouldn't the caseId exist before the session is started?

Reply:
Yes.. I'm storing the caseId in the session part of the state tree because the session was already:
session: { caseId: string, sessionId: string, date: Date }